### ISSUE TITLE
Question: discord api request, user bot

### ISSUE THREAD

**Issue by sequencerr** [erictik/midjourney-client#107]
can discord api be suspicious when sending api request with parameters in order different from sending it from discord interface?
can it be suspicious about sending nonce other type from Snowflake-id?

**Comment by sequencerr** [comment]
it also sends `multipart/form-data` instead of `application/json` for "create command" interaction

**Comment by zcpua** [comment]
there should be no problem.
usually the backend use both form-data and json

**Comment by sequencerr** [comment]
> there should be no problem. usually the backend use both form-data and json

yes, but request sent from discord interface (client) is in type of form-data (in my case)
